src/components/Header/Menu.tsx

```diff
@@ -0,0 +1,122 @@
+import { useState } from "react";
+import { Box, Button, Container, Text } from "@chakra-ui/react";
+import { v4 as uid } from "uuid";
+import Products from "./MenuItems/Products";
+import { CloseIcon } from "@chakra-ui/icons";
+import Services from "./MenuItems/Services";
+import Extras from "./MenuItems/Extras";
+import Support from "./MenuItems/Support";
+import About from "./MenuItems/About";
+
+const links = [
```

Comment:
Can you move this from here? It's making the component really huge. should be put in a different file and exported here.